src/System.Net.Requests/tests/GlobalProxySelectionTest.cs

```diff
@@ -3,47 +3,18 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
-
 using Xunit;
 
 namespace System.Net.Tests
 {
     public class GlobalProxySelectionTest : RemoteExecutorTestBase
     {
-        private class MyWebProxy : IWebProxy
-        {
-            public MyWebProxy() { }
-
-            public ICredentials Credentials
-            {
-                get
-                {
-                    throw new NotImplementedException();
-                }
-
-                set
-                {
-                    throw new NotImplementedException();
-                }
-            }
-
-            public Uri GetProxy(Uri destination)
-            {
-                throw new NotImplementedException();
-            }
-
-            public bool IsBypassed(Uri host)
-            {
-                throw new NotImplementedException();
-            }
-        }
-
         [Fact]
         public void Select_Success()
         {
             RemoteInvoke(() =>
             {
-                var myProxy = new MyWebProxy();
+                var myProxy = new DummyWebProxy();
```

Comment:
I'm not talking about a dummy implementation.  There's already a fairly generic WebProxy class that exists... I'm saying let's use it rather than adding a new file with a new dummy type... just use the type that already exists.